src/pages/UnlockedWallet/AddressesPage/index.tsx

```diff
@@ -98,7 +101,14 @@ const AddressesPage = () => {
           </Header>
           <Tabs>
             <TabBar items={tabs} onTabChange={(tab) => setCurrentTab(tab)} activeTab={currentTab} />
-            <TabContent>{tabs.find((t) => t.value === currentTab.value)?.component}</TabContent>
+            <TabContent>
+              {
+                {
+                  addresses: <AddressesTabContent />,
+                  contacts: <ContactsTabContent />
+                }[currentTab.value]
+              }
+            </TabContent>
```

Comment:
NIT: Should we extract this? 🙃 

```ts
const currentTabComponent = {
  addresses: <AddressesTabContent />,
  contacts: <ContactsTabContent />
}[currentTab.value]

Reply:
Hm, I actually prefer it in the render function for one main reason: I don't have to scroll the file up and down when I am navigating the JSX of the component. If I get to line 108 and then see a variable then I have to look for the definition of the variable somewhere above... Which is how we had it before and it annoyed me a few times.

I don't have a strong opinion about this, if it's more important for you to do it otherwise I can change it.

Reply:
Ok. Well, finding the constant is one cmd + click away. It's the same with callback (like `onClick` methods etc.), everything could be in the render function 🙃 
Kidding, I'm fine with that! Closing.